### ISSUE TITLE
Custom Aspect Ratio 

### ISSUE THREAD

**Issue by proyo-code** [a914-gowtham/Android-video-trimmer#37]
if possible to add custom aspect ratios 

https://imgur.com/xCQgHkG
https://imgur.com/IGzlx2z

**Comment by a914-gowtham** [comment]
Yes video cropping would be great if I add. I will take a look at that however I'm not sure about that

**Comment by proyo-code** [comment]
do you have an update with this ?

**Comment by a914-gowtham** [comment]
sorry @proyo-code I can't work on this now.
